nyc_taxi/anaconda-project.yml

```diff
@@ -18,22 +18,22 @@ channels:
 - defaults
 
 packages: &pkgs
-- python=3.8
-- notebook<7
-- ipykernel=5.3.4
-- bokeh<2.4
-- datashader=0.12.1
-- fastparquet=0.5.0
-- holoviews=1.14.4
-- numpy<1.21
-- pandas<1.4
-- panel=0.11.3
-- python-snappy=0.6.0
-- tornado=6.1
-- xarray=0.18
-- numba<0.55
-- dask<2.11
-- jinja2<3
+- python=3.11
+- notebook
+- ipykernel
+- bokeh>=3.3.4
+- datashader
+- fastparquet
+- holoviews
```

Comment:
```suggestion
- pyviz/label/dev::holoviews>=1.19.0a0
```

Try this.

Reply:
It's giving an error when i try to re-lock the envs:

```
Channels:
 - pyviz
 - defaults
Platform: osx-arm64
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed

PackagesNotFoundError: The following packages are not available from current channels:

  - holoviews==1.19.0a0=py_0

Current channels:

  - https://conda.anaconda.org/pyviz
  - defaults

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


missing requirement to run this project: The project needs a Conda environment containing all required packages.
  '/Users/mac/Documents/development/holoviz-topics-examples/nyc_taxi/envs/default' doesn't look like it contains a Conda environment yet.
'/Users/mac/Documents/development/holoviz-topics-examples/nyc_taxi/envs/default' doesn't look like it contains a Conda environment yet.
Locking failed, re-set old lock file
TaskError - taskid:lock:nyc_taxi
PythonAction Error
Traceback (most recent call last):
  File "/Users/mac/Documents/development/holoviz-topics-examples/dodo.py", line 863, in lock_project
    subprocess.run(
  File "/Users/mac/anaconda3/envs/examples-gallery-manage/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['anaconda-project', 'lock', '--directory', 'nyc_taxi']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/mac/anaconda3/envs/examples-gallery-manage/lib/python3.10/site-packages/doit/action.py", line 461, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "/Users/mac/Documents/development/holoviz-topics-examples/dodo.py", line 875, in lock_project
    shutil.move(oldlockf, lockf)
  File "/Users/mac/anaconda3/envs/examples-gallery-manage/lib/python3.10/shutil.py", line 816, in move
    os.rename(src, real_dst)
TypeError: rename: src should be string, bytes or os.PathLike, not NoneType

```

Reply:
It should use pyviz/label/dev and not pyviz. Try to update the main channel to it.

Reply:
Thanks. It worked now.